### PR TITLE
Headings h1-h6

### PR DESCRIPTION
**Pull request by swese44**
Adds a Heading component. Renders as h1-h6, with size override option to render as h1-h6 or plain text.

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

File: src/components/Heading/enums.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+export type HeadingLevel = '1' | '2' | '3' | '4' | '5' | '6';
+export type HeadingSize = HeadingLevel | 'none';
```
Comment (swese44) [inline src/components/Heading/enums.ts]: @unindented do you think these options make sense?

Comment (unindented) [reply src/components/Heading/enums.ts]: They made sense to me.

Comment (unindented) [reply src/components/Heading/enums.ts]: I guess why `none` instead of `null | undefined`?

Comment (swese44) [reply src/components/Heading/enums.ts]: default `undefined` value would use the current HeadingLevel style, so the unstyled text option needs an actual value. i think it's fine but it's a little odd

**Pull request merged**